package.json

```diff
@@ -39,6 +39,8 @@
   "license": "MIT",
   "dependencies": {
     "axios": "0.27.2",
+    "commander": "^9.3.0",
```

Comment:
Looks like this is unused.

Reply:
Sorry, I forgot to take it out, comes from my new CLI that I use to test.